packages/devtools-launchpad/webpack.config.js

```diff
@@ -57,21 +62,6 @@ module.exports = (webpackConfig, envConfig, options = {}) => {
     loader: "url-loader"
   });
 
-  // Add resolveLoader for ./node_modules to fix issues when synlinked.
-  webpackConfig.resolveLoader = webpackConfig.resolveLoader || {};
-  webpackConfig.resolveLoader.modules = webpackConfig.resolveLoader.modules || [];
-  webpackConfig.resolveLoader.modules.push("node_modules");
-  webpackConfig.resolveLoader.modules.push(path.resolve(__dirname, "./node_modules"));
```

Comment:
this symlinking stuff broke a lot of things:

* we should ignore symlinking the launchpad
* try and get symlinking working later for source-editor or components...